mythril/laser/ethereum/svm.py

```diff
@@ -73,12 +74,18 @@ def sym_exec(self, main_address=None, creation_code=None):
             created_account = self.execute_contract_creation(creation_code)
             logging.info("Finished contract creation, found {} open states".format(len(self.open_states)))
 
+            # Reset code coverage
+            self.coverage = {}
+
             self.time = datetime.now()
             logging.info("Starting message call transaction")
             self.execute_message_call(created_account.address)
 
         logging.info("Finished symbolic execution")
         logging.info("%d nodes, %d edges, %d total states", len(self.nodes), len(self.edges), self.total_states)
+        for code, coverage in self.coverage.items():
+            cov = reduce(lambda sum_, val: sum_ + 1 if val else sum_, coverage[1]) / float(coverage[0]) * 100
+            logging.info("Achieved {} coverage for contract".format(cov))
```

Comment:
It might be a good idea to mention the contract name or something related to it for the coverage to identify which contract it relates to.

Reply:
The code does not necessarily belong to a contract. I changed the message for now to print the code it executed. 

This is more of an intermediate implementation of coverage reporting, and more sophisticated implementation should be done in a separate issue/pr imo